runtime/src/main/java/org/corfudb/util/MetricsUtils.java

```diff
@@ -31,43 +27,30 @@
     private static final MetricSet metricsJVMMem = new MemoryUsageGaugeSet();
     private static final MetricSet metricsJVMThread = new ThreadStatesGaugeSet();
 
-    private static Properties metricsProperties = new Properties();
+    private static int interval;
```

Comment:
why is this static ?

Reply:
One of the properties that similar to metricsCollectionEnabled, and so on being used by other static methods of this util class. Mostly replaces: (private static Properties metricsProperties = new Properties();)